lib/themes/dosomething/paraneue_dosomething/js/campaign/OrganDonation.js

```diff
@@ -137,6 +139,13 @@ class OrganDonation extends Takeover {
     this.loadScreen(this.$shareScreen);
 
     // @TODO - Handle submission of "done" button that posts to our db info about the submission being done.
+    // This happens whether they hit "done" or "skip" since at this point those are the same
```

Comment:
I'm not sure these are the same....would we want to not show the modal anymore if they chose to skip it? I mean we could cause it would be annoying, but they could just skip out of the form for other reasons and we would want to show the modal experience until they actually go through the flow. Could be a product question.